converter/internal/staticconvert/testdata/promtail_prom.river

```diff
@@ -0,0 +1,130 @@
+discovery.consul "name_jobName" {
```

Comment:
This component exists twice and the label is an exact match between prometheus and promtail. I want to call this out so we can decide how to move forward.

Reply:
Brainstorm:
- I think we may need to do something where when adding blocks we check for uniqueness right them on the builder.File. This would require exposing more of that API publicly than is exposed today.

Reply:
I added a todo on a test that demonstrates this

Reply:
I appended `metrics_` or `logs_` to labels to prevent collision between the 2 and provide clearer output. There is 1 edge case left that is now documented in code. A detector on the final output might be a way to go to provide a diag if we detect a collision after completed conversion.